modules/apps/object/object-test-util/src/main/java/com/liferay/object/test/util/ObjectDefinitionTestUtil.java

```diff
@@ -82,6 +92,16 @@ public static ObjectDefinition addCustomObjectDefinition(
 			ObjectDefinitionConstants.STORAGE_TYPE_DEFAULT, objectFields);
 	}
 
+	public static ObjectDefinition addCustomObjectDefinition(
+			long objectFolderId, boolean enableLocalization, String name,
+			List<ObjectField> objectFields, User user)
+		throws Exception {
```

Comment:
I'm not sure if you need this method. Can you avoid adding a new method and using the existing ones?